docs/features/record-routing.mdx

```diff
@@ -0,0 +1,177 @@
+        // rest of metadata
+    },
+    "payload": {
+        "after": {
+            "name": "Adam"
+        }
+    }
+    // other record fields
+}
+```
+
+The PostgreSQL destination connector will check the value in
+the `opencdc.collection` metadata field. Based on that, it will write John's
+information to the `employees_finance` table and Adam's information to
+the `employees_legal` table.
+
+If the `opencdc.collection` metadata field is missing, the record will then be
+written to the default table, which is `employees` in our case.
+
+## Using a `filter` processor with a condition
+
+In some cases, records from a single source collection need to be written to
+multiple connectors. Even if a destination connector can write to multiple
+collections, that feature sometimes cannot be used because the destinations have
+very different configurations. For example, the records may need to be written
+to different databases or different systems altogether.
+
+In scenarios like these, we can route records to multiple destination connectors
+using the built-in [`filter` processor](/docs/processors/builtin/filter) with
+a [condition](/docs/processors/conditions/).
+
+The advantage of this approach is that there will still be only one source,
+which means that we will read the data only once.
+
+### Example: Route data from a single Kafka topic to different PostgreSQL databases
+
+Let's assume we have a pipeline that reads employee data from a topic. The
+employees from the finance department (identified by `department: finance` in
+the metadata) need to be written to one PostgreSQL database. Employees from the
+legal department (identified by `department: legal` in the metadata) need to be
+written into a different database.
+
+```yaml
+---
+version: 2.2
+pipelines:
+  - id: example-pipeline
+    status: running
+    name: example-pipeline
+    description: 'Kafka to PostgreSQL'
+    connectors:
+      - id: kafka-source
+        type: source
+        plugin: "builtin:kafka"
+        name: kafka-source
+        settings:
+          servers: "localhost:9092"
+          topics: "employees_finance,employees_legal"
+
+      - id: finance
+        type: destination
+        plugin: "builtin:postgres"
+        name: finance
+        settings:
+          url: "postgresql://username:password@localhost/finance-db?sslmode=disable"
+          table: "employees"
+        processors:
+        - id: employees_finance_only
+          plugin: "filter"
+          condition: `{{index .Metadata "department" | eq "finance" | not}}`
+     
+      - id: legal
+        type: destination
+        plugin: "builtin:postgres"
+        name: legal
+        settings:
+          url: "postgresql://username:password@localhost/legal-db?sslmode=disable"
+          table: "employees"
+        processors:
+        - id: employees_legal_only
+          plugin: "filter"
+          condition: `{{index .Metadata "department" | eq "legal" | not}}`
```

Comment:
Btw, since `department` is a valid identifier, we could take the shortcut and use `.Metadata.department`. ~I think it should work~ [I tried it out](https://go.dev/play/p/j6VLIS5neqY).
```suggestion
          condition: `{{ eq .Metadata.department "legal" | not }}`
```

Reply:
This one slipped through the cracks @hariso ☝️